### ISSUE TITLE
Error installing with pip?

### ISSUE THREAD

**Issue by dsachs** [Kitware/tangelo#547]
Hi,
I'm on OSX El Capitan, running both system Python 2.7 and homebrew-installed Python 3.5. I have both a 2.7 pip and a 3.5 pip3 installed.

I tried installing tangelo via pip for both 2.7 and 3.5, and neither worked. Each failed with the same error below. Am I doing something wrong?

Here's pip3/Python 3.5:
```
Collecting tangelo
  Using cached tangelo-0.10.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/gx/r62_lrxx39d0jbwp3x9q5c7c0000gn/T/pip-build-w3_pe5kl/tangelo/setup.py", line 24, in <module>
        package_data={"tangelo": rcollect("tangelo/pkgdata")},
      File "/private/var/folders/gx/r62_lrxx39d0jbwp3x9q5c7c0000gn/T/pip-build-w3_pe5kl/tangelo/setup.py", line 8, in rcollect
        return sum(map(lambda x: map(lambda y: x[0] + "/" + y, x[2]), os.walk(path)), [])
    TypeError: can only concatenate list (not "map") to list
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/gx/r62_lrxx39d0jbwp3x9q5c7c0000gn/T/pip-build-w3_pe5kl/tangelo/
```

And here's pip with Python 2.7:
```
Collecting tangelo
  Using cached tangelo-0.10.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/gx/r62_lrxx39d0jbwp3x9q5c7c0000gn/T/pip-build-watt2y9v/tangelo/setup.py", line 24, in <module>
        package_data={"tangelo": rcollect("tangelo/pkgdata")},
      File "/private/var/folders/gx/r62_lrxx39d0jbwp3x9q5c7c0000gn/T/pip-build-watt2y9v/tangelo/setup.py", line 8, in rcollect
        return sum(map(lambda x: map(lambda y: x[0] + "/" + y, x[2]), os.walk(path)), [])
    TypeError: can only concatenate list (not "map") to list
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/gx/r62_lrxx39d0jbwp3x9q5c7c0000gn/T/pip-build-watt2y9v/tangelo/
```

Thanks very much for your help,

David

**Comment by dsachs** [comment]
Looks like someone forked and fixed for python 3:
https://github.com/giantoak/tangelo/commit/a1dc375758338c208b7a782fcb6a1079b3f6da70

**Comment by waxlamp** [comment]
Thanks for the bug report, @dsachs. Are you good to go with @giantoak's fix for the time being?

**Comment by dsachs** [comment]
Roni,

Thanks for the quick reply, much appreciated. Sorry for the delay in
getting back to you.

I actually got it to work, as I realized this morning that in my bug report
I had been using a version of pip that was actually Python 3, not Python 2.
With the Python2 version, installation of Tangelo 0.6.1 (the version I
needed) worked fine.

FYI, I'm installing Tangelo to use mitie-trainer (https://github.com/Sotera/
mitie-trainer).

Thanks and apologies again,

David
